app/_locales/en/messages.json

```
@@ -272,6 +272,15 @@
   "confirmed": {
     "message": "Confirmed"
   },
+  "confusableUnicode": {
+    "message": "'$1' is similar to '$2'. Be careful!"
+  },
+  "confusableZeroWidthUnicode": {
+    "message": "Zero-width character found. Be careful!"
+  },
```

Our latest designs preserve these tooltips, but without the "Be careful!" part:

```suggestion
  "confusableUnicode": {
    "message": "'$1' is similar to '$2'"
  },
  "confusableZeroWidthUnicode": {
    "message": "Zero-width character found"
  },
```